test/packages/EAStdC/CMakeLists.txt

```diff
@@ -2,11 +2,25 @@
 # Copyright (C) Electronic Arts Inc.  All rights reserved.
 #-------------------------------------------------------------------------------------------
 cmake_minimum_required(VERSION 3.1)
+project(EAStdC CXX)
 
 #-------------------------------------------------------------------------------------------
 # Library definition 
 #-------------------------------------------------------------------------------------------
-file(GLOB EASTDC_SOURCES "source/*.cpp")
+set(HEADER_DIR "include/EAStdC")
+set(SOURCE_DIR "source")
+set(EASTDC_SOURCES
+    ${HEADER_DIR}/EAAlignment.h
+    ${HEADER_DIR}/EADateTime.h
```

Comment:
I prefer to have the GLOB style syntax so that I don't I have to edit build scripts when I add/remove files.  This also mimics the behavior of our internal build system.